notebooks/Datasources.ipynb

```diff
@@ -0,0 +1,393 @@
+    "\n",
+    "It's where you can import your dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-03-20 19:19:58.408 | INFO     | neomaril_codex.datasources:import_dataset:279 - Datasource testeDataSouce import process started! Use the D66c8bc440dc4882bfeff40c0dac11641c3583f3aa274293b15ed5db21000b49 on the `/api/datasets/status` endpoint to check it's status.\n",
+      "2024-03-20 19:19:58.410 | INFO     | neomaril_codex.base:__init__:20 - Loading .env\n",
+      "2024-03-20 19:19:58.415 | INFO     | neomaril_codex.base:__init__:30 - Successfully connected to Neomaril\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<neomaril_codex.datasources.NeomarilDataset at 0x7f6ba4f68040>"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dataset_uri = 'https://storage.cloud.google.com/projeto/arquivo.csv'\n",
+    "\n",
+    "dataset = datasource.import_dataset(\n",
+    "    dataset_uri=dataset_uri,\n",
+    "    dataset_name='meudatasetcorreto'\n",
+    ")\n",
+    "dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### List Datasets\n",
+    "\n",
+    "Use this function to find your datasets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'Id': 'D66c8bc440dc4882bfeff40c0dac11641c3583f3aa274293b15ed5db21000b49',\n",
+       "  'CreationDate': '2024-03-20T22:19:58.433928+00:00',\n",
+       "  'Size': 2558,\n",
+       "  'Name': 'meudatasetcorreto',\n",
+       "  'Origin': 'E2dbf476b85e417cb4fdc325a38ee7575a30b81a82264745b3e3a2d92700bc43'}]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "datasource.list_datasets()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Get Datasets\n",
+    "\n",
+    "Use this function to get your dataset neomaril object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-03-20 19:20:16.688 | INFO     | neomaril_codex.base:__init__:20 - Loading .env\n",
+      "2024-03-20 19:20:16.692 | INFO     | neomaril_codex.base:__init__:30 - Successfully connected to Neomaril\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataset = datasource.get_dataset(dataset_hash='D66c8bc440dc4882bfeff40c0dac11641c3583f3aa274293b15ed5db21000b49')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Get Dataset Status\n",
+    "\n",
+    "Use this function to get dataset status."
```

Comment:
Here you can explain what each status mean

Reply:
I think here you can also get the `Running` status, so you can add the description for this expected output too.